### ISSUE TITLE
Can't sign messages with SegWit Addresses 

### ISSUE THREAD

**Issue by thoro** [goatpig/BitcoinArmory#401]
Hi,

seems that the latest branch testing can't sign Segwit addresses.

Stacktrace:

```
(WARNING) toolsDialogs.py:120 - P2SH address requested
(ERROR) Traceback (most recent call last):
  File "/usr/lib/armory/ui/toolsDialogs.py", line 171, in clearSignMessage
    privateKey = self.getPrivateKeyFromAddrInput()
  File "/usr/lib/armory/ui/toolsDialogs.py", line 131, in getPrivateKeyFromAddrInput
    return wallet.addrMap[addr160].binPrivKey32_Plain.toBinStr()
```

**Comment by goatpig** [comment]
I need more context

**Comment by Thoro** [comment]
Okay, I tried to sign a message via the Tools -> Message Signing/Verification Window

I selected the segwit address and put a message, then click Clearsign Signature.

In the console "(WARNING) toolsDialogs.py:120 - P2SH address requested"  is written.

I enter my passphrase.

Then a message box with "Invalid Address" - "The signing address is invalid" pops up.

after clicking okay the stacktrace from above it printed twice.
